src/objecttypes/fixtures/default_admin_index.json

```diff
@@ -0,0 +1,84 @@
+[
```

Comment:
We already have fixtures located in src/fixtures/demodata.json
Should we put new fixtures into the existing folder?

Reply:
Nvm, it's resolved in https://github.com/maykinmedia/objecttypes-api/pull/24